server/database/models/user.js

```diff
@@ -0,0 +1,17 @@
+export default (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      firstName: DataTypes.STRING,
+      lastName: DataTypes.STRING,
+      userName: DataTypes.STRING,
```

Comment:
I don't see the uniqueness constraint for username and email, how did you define the uniqueness?

Reply:
Uniqueness and non-nullable values are defined in the migrations